config/locales/he.yml

```diff
@@ -257,7 +257,7 @@ he:
       muting: רשימת השתקות
     upload: יבוא
   landing_strip_html: <strong>%{name}</strong> על השרת <strong>%{domain}</strong>. ניתן לעקוב אחריהם או לדבר איתם בעזרת חשבון על כל שרת קהילה ברחבי הפדרציה. ליצירת חשבון, ניתן<a href="%{sign_up_path}">להרשם כאן</a>.
-  landing_strip_signup_html: If you don't, you can <a href="%{sign_up_path}">sign up here</a>.
+  landing_strip_signup_html: אחרת, ניתן <a href="%{sign_up_path}">להרשם כאן</a>.
```

Comment:
Thanks for noticing. I use meld to pop out changed keys, I haven't noticed the change in value.

went over again, found a few more changes, pushed a fix.